### ISSUE TITLE
OpenShift registry must be exposed as insecure registry in /etc/sysconfig/docker

### ISSUE THREAD

**Issue by hferentschik** [projectatomic/adb-utils#65]
The OpenShift registry is now exposed as a route in _openshift_provision_:

```
  registry_host_name="hub.openshift.${ip}.xip.io"
  oc expose service docker-registry --hostname ${registry_host_name}
```

But to be able to push to the registry one also needs to add the route/host to _/etc/sysconfig/docker_.
For now I have to add:

```
/opt/adb/openshift/add_insecure_registry -ip #{REGISTRY_HOST}
```

to my _Vagrantfile_.

I am pretty sure we already discussed the need for this insecure registry entry somewhere. I think in the context the changes to _add_insecure_registry_.

**Comment by hferentschik** [comment]
Can we add the label focus:devex and also mark it as a blocker.

**Comment by kadel** [comment]
Rather than exposing registry as insecure why not configure it properly as secure registry?

In documentation is nice guide how to secure and expose it https://docs.openshift.org/latest/install_config/install/docker_registry.html#exposing-the-registry

**Comment by hferentschik** [comment]
> Rather than exposing registry as insecure why not configure it properly as secure registry?

Sure, I am all up for it. Note, however, that these are 10 additional steps including certificate creation and what not. Definitely more work. Also, last time I tried I got hung up somewhere. To be fair, I moved to insecure quite quickly then w/o much more digging, but it felt for the type of environment we are offering running the registry insecurely was fair enough.
